src/main/java/uk/gov/companieshouse/accounts/association/models/InvitationDao.java

```diff
@@ -4,6 +4,8 @@
 
 import java.time.LocalDateTime;
 
+import static uk.gov.companieshouse.accounts.association.service.AssociationsService.DAYS_SINCE_INVITE_TILL_EXPIRES;
```

Comment:
i would not add this here.
i would either add it to the properties file and reference it in the Dao or add it as a constant in the dao.